proto/greenfield/storage/events.proto

```diff
@@ -68,16 +68,14 @@ message EventUpdateBucketInfo {
     (gogoproto.customtype) = "Uint",
     (gogoproto.nullable) = false
   ];
-  // charged_read_quota_before define the read quota before updated
-  uint64 charged_read_quota_before = 4;
   // charged_read_quota_after define the read quota after updated
-  uint64 charged_read_quota_after = 5;
-  // payment_address_before define the payment address before updated
-  string payment_address_before = 6;
-  // payment_address_after define the payment address after updated
-  string payment_address_after = 7;
+  uint64 charged_read_quota = 4;
+  // payment_address define the payment address after updated
+  string payment_address = 5;
   // visibility defines the highest permission of object.
-  VisibilityType visibility = 8;
+  VisibilityType visibility = 6;
+  // global_virtual_group_family_id defines the gvg family id after migrated.
+  uint32 global_virtual_group_family_id = 7;
 }
```

Comment:
EventCompleteMigrationBucket may need to add `src sp id` field because src sp should gc bucket objects when the complete bucket migrates.

Reply:
Added.